portfolio/src/main/webapp/script.js

```diff
@@ -63,15 +63,53 @@ function showComments() {
         document.getElementById("thecomments").innerHTML += "<p>" + 
         jsoncomments[jsoncomments.length - 1] + "</p";
     })
+    loadComments();
 }
 
 /**
  * Accesses the comments from the datastore and displays them.
  */
 function loadComments() {
     fetch("/data").then(dataHashMap => dataHashMap.json()).then(dataHashMapJson => {
-        for (comment of dataHashMapJson.Comments) {
-            document.getElementById("thecomments").innerHTML += "<p>" + comment + "</p";
+        if (document.getElementById("CommentNumber").options[0].selected === true) { //5
+            for (i = 0; i < 5; i++) {
+                if (dataHashMapJson.Comments.length < i) {
+                    break;
+                } else {
+                    if (i == 0) {
+                        document.getElementById("thecomments").innerHTML = "<p>" + 
```

Comment:
Can you explain a bit more on the reason for distinguishing the first comments from the other?

Reply:
There is "commentsection" and "thecomments"

"thecomments" is like the area where the user can see the already posted comments 

"commentsection" is the area where the user types and posts their own comments.

It actually comes in handy during authentication (which is ironic). But I knew I wanted to make it where the user couldnt post comments until they were logged in. So I kinda wanted to hide the comment section div and still show "thecomments"


Also when I am appending to "thecomments" i want to easily do .getelementbyid

Reply:
Can you deploy your project to cloud? I didn't seem to see "commentsection" in html.
Also, as to following code:
 dataHashMapJson.Comments[len - 1] + "</p";
"</p" doesn't seem to close tag properly, I'm curious how it get's rendered.

Reply:
Yeah I'll deploy it. Also that missing ">" is definitely a mistake lol I'll fix that.